### PR TITLE
[INJIMOB-2452]: Add Credential Issuer Host.

### PR DESCRIPTION
**Pull request by balachandarg-tw**
- Add Credential Issuer ID which holds the value of issuer host.
- New field is used by Consumer(Injiwallet) to append well known end point for wellknown discovery in Inji Wallet side as per [OpenId4VCI](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-10.2.2).

Ticket Number: [INJIMOB-2452](https://mosip.atlassian.net/browse/INJIMOB-2452)

[INJIMOB-2452]: https://mosip.atlassian.net/browse/INJIMOB-2452?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mimoto-issuers-config.json [hunk 1]
```diff
@@ -350,10 +356,11 @@
       "token_endpoint": "https://${mosip.api.public.host}/v1/mimoto/get-token/mockFarmer",
       "proxy_token_endpoint": "https://${mosip.esignet.mock.host}/v1/esignet/oauth/v2/token",
       "qr_code_type": "EmbeddedVC",
+      "credential_issuer": "https://nearly-notable-longhorn.ngrok-free.app",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:359]: We should clean up ngrok from this file

File: mimoto-issuers-config.json [hunk 2]
```diff
@@ -325,10 +330,11 @@
           "token_endpoint": "https://${mosip.api.public.host}/v1/mimoto/get-token/Mock-Fallback",
           "proxy_token_endpoint": "https://${mosip.esignet.mock.host}/v1/esignet/oauth/v2/token",
           "qr_code_type": "OnlineSharing",
+          "credential_issuer": "http://config-server.config-server1",
```
Comment (swatigoel) [inline mimoto-issuers-config.json:333]: Is this the issuer?

**Review by swatigoel — CHANGES_REQUESTED** [review]

File: mimoto-issuers-config.json (hunk 2 above)
Comment (balachandarg-tw) [reply mimoto-issuers-config.json:333]: Not a issuer, it is for fallback scenario just copied from exisitng wellknown base url. Shall we remove all fallback issuers?

File: mimoto-issuers-config.json (hunk 1 above)
Comment (balachandarg-tw) [reply mimoto-issuers-config.json:359]: Updated

File: mimoto-issuers-config.json (hunk 2 above)
Comment (balachandarg-tw) [reply mimoto-issuers-config.json]: Removed fallback issuers from list.

**Review by swatigoel — APPROVED** [review]

**Pull request merged**